Atos20/Rancid-Tomatillos
# Feature/get user ratings

### PR Template:

#### What’s this PR do?  
This branch utilizes fetch to retrieve a user's ratings, and set the correlating state.
 
#### Where should the reviewer start?  
`authenticateUser` in `App.js`
 
#### How should this be manually tested?  
You can console.log either of the fetch promises in the above mentioned function to see that the proper data is being retrieved.
 
#### Any background context you want to provide?  
This will be used to update the movie cards with a user's ratings for each movie.
 
#### What are the relevant tickets?  
Closes #55 

 
#### Screenshots (if appropriate) 
n/a
 
#### Questions: 
n/a
